packages/lab/src/dialog/DialogTitle.tsx

```diff
@@ -26,19 +25,19 @@ interface DialogTitleProps
   /**
    * Displays the Dialog Title in a H2 component
    */
-  title: ReactNode;
+  header: ReactNode;
   /**
    * Displays the Dialog Subtitle in a Label component
```

Comment:
This comment needs to be updated and it should describe what preheader is rather than what component is rendered